### PR TITLE
Update check_mk_agent.linux

### PR DESCRIPTION
**Pull request by Timdebruijn**
fix the cpu count check for aarch64

With the new 64bit OS for example on the Raspberrypi, the cpu count is wrong:

```
root@raspberry:~# uname -m
aarch64
root@raspberry:~# grep -c -E '^processor' </proc/cpuinfo
4
root@raspberry:~# grep -c -E '^CPU|^processor' </proc/cpuinfo
24
```

### CODE REVIEW

File: agents/check_mk_agent.linux [hunk 1]
```diff
@@ -439,7 +439,7 @@ section_mem() {
 }
 
 section_cpu() {
-    if [ "$(uname -m)" = "armv7l" ] || [ "$(uname -m)" = "armv6l" ]; then
+    if [ "$(uname -m)" = "armv7l" ] || [ "$(uname -m)" = "armv6l" ] || [ "$(uname -m)" = "aarch64" ]; then
```
Comment (svenpanne) [inline agents/check_mk_agent.linux:442]: Quick drive-by comment: When we have a 3rd condition, using a `case` might be more appropriate. It's more readable and more efficient (only a single uname call).

**Comment by kain88-de** [comment]
Thank you for your contribution! This change will be merged shortly.
